### PR TITLE
Process 2018

### PR DESCRIPTION
**Pull request by deniak**
That PR adds new profiles introduced by the process 2018:
- amended CR, PR, REC
- superseded REC

It also updates the abbreviation required for obsoleted and rescinded RECs for harmonization with the different types of recommendations.

### CODE REVIEW

File: test/docs/sotd/deprecated2015.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
   <body>
     <div class="head">
       <h1>Simple baseline headers valid document</h1>
-      <h2>W3C Working Draft 15 March 2017</h2>
+      <h2>W3C Working Draft 15 March 2018</h2>
     </div>
     <h2>Status of This Document</h2>
-    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2015/Process-20150901/">1 September 2015 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
+    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2015/Process-20150801/">1 September 2015 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
```
Comment (plehegar) [inline test/docs/sotd/deprecated2015.html]: shouldn't it stay 0901? 0801 returns a 404...

Comment (deniak) [reply test/docs/sotd/deprecated2015.html]: Good catch. Fixed

**Comment by deniak** [comment]
> Should we keep the tree of profiles on README#6-profiles updated, or get rid of it? (I personally find it useful as a reference, but I'm okay if you prefer to remove it, @deniak.)

@tripu, I updated the README. Indeed, it can be useful.

> Does this close #722?

Yes, it closes that issue.

**Comment by plehegar** [comment]
https://github.com/w3c/modern-tooling/issues/71

**Pull request merged**